README.md

```diff
@@ -1 +1,38 @@
-# Setup-and-Mobile-Version-skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Project Name
+
+  Setup and Mobile Version Skeleton
+
+
+> Description the project.
+
+The 2 sections of this skeleton project has:
+
+a - the toolbar
+b - the headline section
+
+
+
+## Built With
+
+- HTML
+- CSS
+- linter
+
+
+
+
+## Author
+
+👤 **Pedro Marques**
+
+- GitHub: https://github.com/ppmarq1
+- Twitter: https://twitter.com/ppmarq1
+- LinkedIn: https://br.linkedin.com/in/pedroalmeidamarques/
+
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[ Required Changes ]

- [ ] You're doing great with the project so far. But would love for you to consider completing your README structure to match this [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md) with your personal contents. Please consider placing all headings in the template and placing your contents relating to the project.👍